### DISCUSSION TITLE
Just sharing utility tool for ALS

### DISCUSSION THREAD

**Discussion by oivio** [Sixze/ALS-Refactored#109] [General]
[ALS-Refactored_utility.zip](https://github.com/Sixze/ALS-Refactored/files/9081620/ALS-Refactored_utility.zip)
![image(71)](https://user-images.githubusercontent.com/38628851/178208378-088b21ca-9cb2-4835-9df3-cf3ae9618ae2.png)

Sharing a simple utility tool that adds ALS VB and Sockets to the new Skeleton mesh. 

PS: Thank you for your amazing work!!!

If I could just recommend is to also work on separating mantle and camera. Something like ALS Community has it. This would make an even much more modular ALS system.

**Comment by Sixze** [comment]
Thanks for the tools, I'll take a look at them in my spare time and probably integrate them into the plugin.

The camera is mostly separate from the character, it only reads its state, but this can be changed using the C++ interfaces.

Mantling and other locomotion actions mostly exist as examples that you shouldn't use directly in your game. I'm thinking about redoing them using the Gameplay Ability System, but that could make an already complex plugin even more complex.
